packages/preview-middleware/README.md

```diff
@@ -39,6 +40,12 @@ Array of additional application configurations:
 | `client`      | `string` optional              | sap-client parameter                                                                                                                            |
 | `scp`         | `boolean` optional             | If set to true the proxy will execute the required OAuth routine for the ABAP environment on SAP BTP                                            |
 
+### `adp.auth`
```

Comment:
I am against allowing to set user/password in the yaml. It is a file that is committed to git and should not contain such sensitive information.